prime-angular-frontend/src/app/modules/auth/shared/enum/role.enum.ts

```diff
@@ -4,7 +4,10 @@ export enum Role {
   SUPER_ADMIN = 'prime_super_admin',
   READONLY_ADMIN = 'prime_readonly_admin',
 
+  PHSA_LABTECH = 'phsa_eforms_labtech',
+  PHSA_IMMUNIZER = 'phsa_eforms_immunizer_covid19',
```

Comment:
Distinguishing role indicating they are not just a `prime_user`, but specifically part of PHSA